docs/snippets/common/storybook-auto-docs-mdx-file.mdx.mdx

```diff
@@ -0,0 +1,34 @@
+```mdx
```

Comment:
I don't think it needs to block this PR, but I find this a confusing example of combining components into one doc. How is Button related to Page? How about List and ListItem?